.github/workflows/wheels.yml

```diff
@@ -53,10 +53,10 @@ jobs:
       skbuild-configure-options: "-DCUML_BUILD_WHEELS=ON -DDETECT_CONDA_ENV=OFF -DCPM_cumlprims_mg_SOURCE=/project/cumlprims_mg/"
 
       # Always want to test against latest dask/distributed.
-      test-before-amd64: "pip install git+https://github.com/dask/dask.git@main git+https://github.com/dask/distributed.git@main git+https://github.com/rapidsai/dask-cuda.git@branch-22.12"
+      test-before-amd64: "pip install git+https://github.com/dask/dask.git@2022.11.1 git+https://github.com/dask/distributed.git@2022.11.1 git+https://github.com/rapidsai/dask-cuda.git@branch-22.12"
```

Comment:
Might have to re-base your branch with `branch-23.02` to address these unnecessary changes.